include/boost/variant/recursive_wrapper.hpp

```diff
@@ -45,7 +45,11 @@ class recursive_wrapper
     recursive_wrapper(const T& operand);
 
 #ifndef BOOST_NO_CXX11_RVALUE_REFERENCES 
+#ifndef BOOST_VARIANT_NO_RECURSIVE_WRAPPER_POINTER_STEALING
```

Comment:
Please add docs for that macro and also note it near the `empty()` functions.

Reply:
Should I update never-empty guarantee page with a note about recursive variants can be empty?

Reply:
Yes please

Reply:
Done.